src/main/flight/imu.c

```diff
@@ -666,7 +666,7 @@ void imuUpdateAttitude(timeUs_t currentTimeUs)
 
 bool isImuReady(void)
 {
-    return sensors(SENSOR_ACC) && gyroIsCalibrationComplete();
+    return sensors(SENSOR_ACC) && STATE(ACCELEROMETER_CALIBRATED);
```

Comment:
I think it should be rather extended to `sensors(SENSOR_ACC) && gyroIsCalibrationComplete() && STATE(ACCELEROMETER_CALIBRATED)`